src/mad_spark_alt/core/robust_orchestrator.py

```
@@ -16,10 +16,10 @@
     ThinkingAgentInterface,
     ThinkingMethod,
 )
-from .timeout_wrapper import TimeoutManager, with_timeout, TimeoutError
 from .robust_json_handler import safe_parse_ideas_array
-from .smart_orchestrator import SmartQADIOrchestrator, SmartQADICycleResult
+from .smart_orchestrator import SmartQADICycleResult, SmartQADIOrchestrator
 from .smart_registry import SmartAgentRegistry
+from .timeout_wrapper import TimeoutError, TimeoutManager, with_timeout
```

Ruff (0.11.9)

`.timeout_wrapper.with_timeout` imported but unused

Remove unused import: `.timeout_wrapper.with_timeout`

(F401)